amf-client/shared/src/test/resources/org/raml/parser/examples/music-api/output.txt.j

```diff
@@ -0,0 +1,27 @@
+Model: file://amf-client/shared/src/test/resources/org/raml/parser/examples/music-api/input.raml
```

Comment:
Why was this file created? By the extension i would asume this is not being used.

Reply:
removed